src/rdkafka.h

```diff
@@ -4827,8 +4861,24 @@ struct rd_kafka_group_member_info {
         void *member_assignment;    /**< Member assignment (binary),
                                      *    format depends on \p protocol_type. */
         int member_assignment_size; /**< Member assignment size in bytes */
+        rd_kafka_topic_partition_list_t
+            *member_assignment_toppars; /**< Member assignment topic partitions
+                                         */
 };
 
+/**
+ * @brief Consumer group states
+ */
+typedef enum {
+        RD_KAFKA_CGRP_STATE_UNKNOWN              = 0,
```

Comment:
CONSUMER_GROUP

"cgrp" is an internal librdkafka thing.

Reply:
done